CorsixTH/Lua/movie_player.lua

```diff
@@ -45,6 +46,10 @@ local calculateSize = function(me)
   local screen_w, screen_h = me.app.config.width, me.app.config.height
   local native_w = me.moviePlayer:getNativeWidth()
   local native_h = me.moviePlayer:getNativeHeight()
+  if native_w == 320 and native_h == 200 then
```

Comment:
I would've preferred detection of the movie itself to be played but this works too.